Dockerfile.must-gather

```diff
@@ -2,4 +2,4 @@ FROM quay.io/openshift/origin-cli:4.19
 
 COPY must-gather/collection-scripts/* /usr/bin/
 
-ENTRYPOINT /usr/bin/gather
+ENTRYPOINT ["/usr/bin/gather"]
```

Comment:
this is fine, but doesn't clarify why the exec form is better (I'm fine with both TBH)

Reply:
I added a link to the official docs that explains why

Reply:
Yes, I saw it, thanks. Could you please make it explicit which of the reasons apply to us?
1. just use the preferred form because both are valid for our use case (a bit light, but OK)
2. prevent the shell process to be run
3. avoid any possible issue with signal delivery
4. streamline `ENTRYPOINT` and `CMD` interaction and avoid future bugs

all the above?
something else?

Reply:
`1` the default approach is that we should always follow best-practice unless we have a well defined reason why not to.

Please correct me if i'm wrong here